### ISSUE TITLE
[Feature request] Bee house textures do not generate from texture packs

### ISSUE THREAD

**Issue by myrmex1** [GrowthcraftCE/Growthcraft-1.7#381]
Would it be possible to make Growthcraft dynamically generate bee house textures from the textures of the planks? Re-texturing every single one just to get texture support is a pretty daunting task.

**Comment by IceDragon200** [comment]
tl;dr
Yes, we could, but then it would look absolutely terrible.

@Qqhelethh: Have you ever seen the original bee box textures?

Oh right, there was only 4 (each side) (1 box.), which was a Oak Plank texture.

By the way, it looked horrible and many, MANY times the texture threw me off because I couldn't identify the edges of the object.

So I changed the texture. Gave it [definite outlines](https://dl.dropboxusercontent.com/u/72618186/screenshots/minecraft/growthcraft/beebox/2015-10-13_18.40.23.png), that way you could pick the block out among planks ("Oh, that's where I left it")

Later down the monotonous Oak texture got to me, so I did the minecraft wood variants.
After playing around in Gimp for half a day trying to figure out how to effectively recolor them, I figured out a process to recolor them in roughly 30 seconds per texture.

Later we started testing integration with Biomes'O'Plenty, and I really liked the wood types there, so I created the textures for them as well.

And did the same for Natura, and Thaumcraft, all by hand (except one set that was done by @Sunconure11)

So yes, certainly I could have generated all of them by grabbing the plank textures from existing mods/packs, but I wanted them to have a special feel, not just a reshaped block with a plank texture slapped on it.

And from what I've heard, people really enjoyed the new look.

In all this took roughly 2 days (if you compress all the texture time together, and the time wasted to figure it out the first time)

**Comment by myrmex1** [comment]
Thanks for the response. Your reasoning makes sense, and you're right that the new textures look much better. I guess I will work on texturing them for the texture pack I am using.
